source/module_cell/read_atoms.cpp

```diff
@@ -415,16 +415,17 @@ bool UnitCell::read_atom_positions(std::ifstream &ifpos, std::ofstream &ofs_runn
 			// start magnetization
 			//=======================================
 			ModuleBase::GlobalFunc::READ_VALUE(ifpos, atoms[it].label);
-			bool found = false;
+			bool sequence_match = false;
 			for(int it2=0; it2<ntype; it2++)
 			{
-				if( this->atoms[it].label == this->atom_label[it] )
-				{
-					found = true;
+				if( this->atoms[it].label == this->atom_label[it2] )
+				{	
+					if (it2 == it) sequence_match = true;
```

Comment:
I can't understand modifies in this PR, it looks for-loop of it2 is useless. 
And this PR looks no changes actually.